services/frontend/react_app/src/pages/home.tsx

```diff
@@ -10,6 +24,16 @@ import {Locations} from "../types/location.ts";
 import {LocationSelect} from "../components/common/locationSelect/locationSelect.tsx";
 import {useLogs} from "../hooks/useLogs.tsx";
 import {NotificationResponse} from "../types/notifications.ts";
+import {useGetRequestArrival} from "../hooks/useGetRequestArrival.tsx";
+import Countdown, {CountdownRenderProps} from "react-countdown";
+
+const countdownRenderer = ({minutes, seconds, completed, driverName}: CountdownRenderProps & { driverName: string }) => {
+    if (completed) {
+        return <Text>{driverName} arrived</Text>;
+    } else {
+        return <Text as="b">The driver {driverName} will arrive in {minutes.toString().padStart(2, "0")}:{seconds.toString().padStart(2, "0")}</Text>;
```

Comment:
We also want to put the Driver's Plate (T702061C) here. The message should be:
`The driver {driverName} (plate) will arrive in X minutes`